scripts/utils/trading_strategy_helpers.js

```diff
@@ -109,6 +109,18 @@ module.exports = function (web3 = web3, artifacts = artifacts) {
     return ownerAddresses.length == 1 && ownerAddresses[0] == masterAddress
   }
 
+  /**
+   * Checks that a bracket has not yet made any orders
+   */
+  const checkNoOrdersPlaced = async function (bracket, exchange) {
+    const orders = await exchange.getEncodedUserOrders.call(bracket)
+    if (orders === null) {
+      return true
+    } else {
+      return false
+    }
+  }
```

Comment:
Wouldn't this be easier if we just check the `orderIndex` of this user?

Reply:
What kind of orderIndex? Are you talking about the length of the orders vector? In this case, I can not get a length of an null object.

Reply:
I see. I guess the contract does not expose a public vie like `getOrderIndex(address)`. I wonder if this might be a good thing to include in v2.

Reply:
I think maybe we could/should make this function more sophisticated by checking that there are no "valid" orders. At least this way, we could reuse brackets instead of abandoning them.